packages/amplify-category-auth/resources/auth-custom-resource/hostedUIProviderLambda.js

```diff
@@ -1,104 +1,100 @@
 const response = require('cfn-response');
 const aws = require('aws-sdk');
 const identity = new aws.CognitoIdentityServiceProvider();
-exports.handler = (event, context, callback) => {
+const ssm = new aws.SSM();
+
+exports.handler = async(event, context) => {
   try {
     const userPoolId = event.ResourceProperties.userPoolId;
+    let responseData;
+    const parameter = await ssm.getParameter({
```

Comment:
does this work if the env vars isn't set and also if the value isn't present in SSM?

Reply:
this wont happen as amplify push , env add , update auth and auth adds the secret to parameter store if hostedUI is true.

Reply:
Just as long as we're sure that there won't be a case where this env var is set but the parameter isn't or a case where this lambda would execute without the env var